common/lib/xmodule/xmodule/js/src/video/10_main.js

```diff
@@ -138,6 +140,10 @@
                     initialize(state, element);
                 }
 
+                if (autoAdvanceEnabled) {
+                    mainVideoModules.push(VideoAutoAdvanceControl);
+                }
```

Comment:
Please place this statement right after where the vars are defined instead (line 92). There is code right above this statement using `state`, which has `mainVideoModules` in it.